src/components/ChartContainer/D3Container.js

```diff
@@ -147,7 +147,9 @@ function D3Container({ dashboardState, dashboardActions, store }) {
   const handleMeasureChange = (event) => {
     let newSelectedMeasures;
     if (event.target.checked) {
-      newSelectedMeasures = selectedMeasures.concat(event.target.value);
+      newSelectedMeasures = event.target.value === 'all'
+        ? store.currentResults.map((result) => result.measure)
+        : selectedMeasures.concat(event.target.value);
       setSelectedMeasures(newSelectedMeasures);
     } else {
       newSelectedMeasures = selectedMeasures.filter(
```

Comment:
Currently when I select the top checkbox when everything is selected, nothing happens. I'm assuming everything is supposed to be unchecked. If that is the case just throw in this on line 154 between the if and the else

`else if (event.target.value === 'all') {
      newSelectedMeasures = selectedMeasures.filter(
        (result) => result === event.target.value,
      );
      setSelectedMeasures(newSelectedMeasures);
    }`